components/automate-ui/src/app/modules/policy/details/policy-details.component.ts

```diff
@@ -46,7 +46,7 @@ export class PolicyDetailsComponent implements OnInit, OnDestroy {
   ) {  }
 
   ngOnInit(): void {
-    this.layoutFacade.showSettingsSidebar();
+    this.layoutFacade.showSidebar('settings');
```

Comment:
I would prefer to see the arguments to `showSidebar` come from an enum, so this would look like:
```suggestion
    this.layoutFacade.showSidebar(Sidebar.Settings);
```
That makes it much less fragile (due to typos, etc.)

Reply:
@msorens Thanks for brining that up. I would agree. I'll add that! 😊